src/app/examples/form/form.model.ts

```diff
@@ -0,0 +1,14 @@
+export interface Form {
+  username: string;
+  password: string;
+  email: string;
+  description: string;
+  requestGift: boolean;
+  age: number;
```

Comment:
Could it be that this is a leftover from the rebase?
I think this can be removed?